### PR TITLE
Maint/black isort

### PR DESCRIPTION
**Pull request by cemathey**
`black` and `isort` every Python file and remove unused imports.

I didn't build/test this but I'm trusting in Pylance to not lead me astray.

### CODE REVIEW

**Review by FlorianSW — APPROVED** [review]
didn't go through the files itself, but as this is an automated process, just approving it so you can merge once merge conflicts are resolved 👍

**Comment by cemathey** [comment]
I had weird issues after fixing merge conflicts so I messed something up. Wasn't worth the time to try to diagnose/fix, just going to delete this branch, make a fresh one off master, black/isort that and re-submit.

**Pull request closed without merging**
